sdk/ai/ai-projects/review/ai-projects.api.md

```diff
@@ -98,22 +98,18 @@ export interface AgentsOperations {
     createThreadAndRunStreaming: (assistantId: string, options?: Omit<CreateAndRunThreadOptions, "assistant_id">, requestParams?: OptionalRequestParameters) => AsyncIterable<AgentStreamEventMessage>;
     createVectorStore: (options?: VectorStoreOptions, requestParams?: OptionalRequestParameters) => Promise<VectorStoreOutput>;
     deleteAgent: (assistantId: string) => Promise<AgentDeletionStatusOutput>;
-    // Warning: (ae-forgotten-export) The symbol "DeleteFileParameters" needs to be exported by the entry point index.d.ts
-    deleteFile: (fileId: string, options?: DeleteFileParameters) => Promise<FileDeletionStatusOutput>;
+    deleteFile: (fileId: string, requestParams?: RequestParameters) => Promise<FileDeletionStatusOutput>;
```

Comment:
change to OptionalRequestParameters

Reply:
I updated to now use OptionalRequestParameters